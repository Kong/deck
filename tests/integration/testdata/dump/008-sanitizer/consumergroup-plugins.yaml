_format_version: "3.0"
consumer_groups:
- id: 28fa8375-8ed4-4678-8fdd-1909beefb36f
  name: silver
  tags:
    - tag1
    - tag3
  plugins:
  - id: e01b2c4c-9378-4f12-80f1-60bb146435f2
    name: rate-limiting
    config:
      policy: local
      limit_by: consumer
      hour: 1
      sync_rate: -1
consumers:
- groups:
  - name: silver
  id: b4e88f37-71ff-483c-af6c-686638d843f6
  username: bar
- id: 9dd504de-919b-4d1b-b607-5d82d8041ba5
  username: baz
