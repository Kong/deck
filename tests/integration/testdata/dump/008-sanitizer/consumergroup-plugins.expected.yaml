_format_version: "3.0"
consumer_groups:
- id: 28fa8375-8ed4-4678-8fdd-1909beefb36f
  name: 6b86b58aa18e1ce01b7d24652f8cf260227543880454c04b79552778e223dee3
  plugins:
  - config:
      day: null
      error_code: 429
      error_message: 19874afab7846c1cd7babc161103715662cd7deadbbebfca35f6c2fd3cad88c0
      fault_tolerant: true
      header_name: null
      hide_client_headers: false
      hour: 1
      limit_by: consumer
      minute: null
      month: null
      path: null
      policy: local
      redis:
        database: 0
        host: null
        password: null
        port: 6379
        server_name: null
        ssl: false
        ssl_verify: false
        timeout: 2000
        username: null
      redis_database: 0
      redis_host: null
      redis_password: null
      redis_port: 6379
      redis_server_name: null
      redis_ssl: false
      redis_ssl_verify: false
      redis_timeout: 2000
      redis_username: null
      second: null
      sync_rate: -1
      year: null
    id: e01b2c4c-9378-4f12-80f1-60bb146435f2
    name: rate-limiting
  tags:
  - cbe2fd20f5a2488c30f5fb6604a2efebf8aa1d7647539065fd9c223823346890
  - 15824198f2123fa83b55b55618aacfa9b002f865456f155d3a342db3fdd8756e
consumers:
- groups:
  - id: 28fa8375-8ed4-4678-8fdd-1909beefb36f
    name: 6b86b58aa18e1ce01b7d24652f8cf260227543880454c04b79552778e223dee3
    tags:
    - cbe2fd20f5a2488c30f5fb6604a2efebf8aa1d7647539065fd9c223823346890
    - 15824198f2123fa83b55b55618aacfa9b002f865456f155d3a342db3fdd8756e
  id: b4e88f37-71ff-483c-af6c-686638d843f6
  username: 900474b796208023ea49b5966dfdedca1bccd9c6ea6ee5802c6b97a2f462f741
- id: 9dd504de-919b-4d1b-b607-5d82d8041ba5
  username: 45a12bf92b4071881eb554f67e7426c6b3456cca7804dd77ec5e36874758b52c
